android/quest/src/main/java/org/smartregister/fhircore/quest/ui/register/RegisterFragment.kt

```diff
@@ -69,6 +74,8 @@ class RegisterFragment : Fragment(), OnSyncListener, Observer<QuestionnaireSubmi
 
   @Inject lateinit var syncListenerManager: SyncListenerManager
 
+  @Inject lateinit var eventBus: EventBus
+
   @VisibleForTesting val appMainViewModel by activityViewModels<AppMainViewModel>()
 
```

Comment:
prefer no line breaks 
```suggestion
  @Inject lateinit var syncListenerManager: SyncListenerManager
  @Inject lateinit var eventBus: EventBus
  @VisibleForTesting val appMainViewModel by activityViewModels<AppMainViewModel>()
```